### ISSUE TITLE
Debug Roslyn Source Code Generators in VS2022 ? 

### ISSUE THREAD

**Issue by ignatandrei** [dotnet/roslyn-sdk#943]
For debug Roslyn Source Code Generators in VS2022 , we have had  first

Debugger.Launch 

and later 

<IsRoslynComponent> true</IsRoslynComponent>

But I cannot figure how to do it in VS2022. What will be the profile ?

**Comment by sharwell** [comment]
I highly recommend debugging source generators by writing unit tests and debugging them from Test Explorer. Debugging a source generator inside of a live Visual Studio instance will be problematic due to their incremental nature and difficulty predicting when they will start.

**Comment by dominikjeske** [comment]
I agree with @sharwell - take a look at https://dominikjeske.github.io/source-generators/ in Testing section
